src/__test__/Home.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Home from '../components/Home';
+
+test('anchor Home snapshot', () => {
+  const tree = renderer.create(<Home />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] This snapshot is failing, kindly update it ![image](https://user-images.githubusercontent.com/62705947/214432797-7baa09bc-334c-4a2f-af48-bc83842e14e1.png)